### PR TITLE
MINOR: A few logging improvements in group coordinator

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

File: core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala [hunk 1]
```diff
@@ -601,10 +600,6 @@ class GroupMetadataManager(brokerId: Int,
               throw new IllegalStateException(s"Unexpected unload of active group $groupId while " +
                 s"loading partition $topicPartition")
           }
-
-          if (!shuttingDown.get())
```
Comment (ijuma) [inline core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala:605]: Do you know why we had this check before?

File: core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala [hunk 2]
```diff
@@ -323,7 +323,7 @@ private[transaction] class TransactionMetadata(val transactionalId: String,
           }
 
         case CompleteAbort | CompleteCommit => // from write markers
-          info(s"transit start ${transitMetadata.txnStartTimestamp}")
+          debug(s"Transaction for $transactionalId begin transit to $toState at ${transitMetadata.txnStartTimestamp}")
```
Comment (ijuma) [inline core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala]: Is there a typo here or is `transit` intentional?

**Review by ijuma — COMMENTED** [review]
Thanks for the PR, just one question and one comment.

File: core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala (hunk 2 above)
Comment (hachikuji) [reply core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala]: We have used the word "transit" intentionally in place of "transition"  in some cases. Actually the log message below seems sufficient to cover this message, so I'm just going to remove this log.

File: core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala (hunk 1 above)
Comment (hachikuji) [reply core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala:605]: I am not sure, but it is weird, right? Why would we be loading during shutdown, and if we are, it seems useful knowing about it in the logs.

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

**Comment by hachikuji** [comment]
Merging to trunk and 0.11.0.

**Pull request closed without merging**